code/game/gamemodes/cult/cult_items.dm

```diff
@@ -192,13 +192,15 @@
 /obj/item/weapon/whetstone/cult
 	name = "eldritch whetstone"
 	desc = "A block, empowered by dark magic. Sharp weapons will be enhanced when used on the stone."
-	icon = 'icons/obj/cult.dmi'
-	icon_state = "cultstone"
+	icon_state = "cult_sharpener"
 	used = 0

```

Comment:
does `used = 0` need to be their as well? It's already '0' in the parent `/obj/item/weapon/whetstone`

Reply:
probably was just copy pasta from the original addition of the subtype